test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -91,60 +106,96 @@ public void VerifyImages_2_1()
         }
 
         private void VerifySdkImage_NewRestoreRun(
-            string appSdkImage, string sdkImageVersion, string netcoreappVersion)
+            string appSdkImage, string sdkImageVersion, string osVariant, string netcoreappVersion)
         {
-            string sdkImage = GetDotNetImage(sdkImageVersion, DotNetImageType.SDK);
-            string buildArgs = GetBuildArgs($"netcoreapp_version={netcoreappVersion}");
-            DockerHelper.Build($"Dockerfile.{DockerHelper.DockerOS.ToLower()}.test", sdkImage, appSdkImage, buildArgs);
+            // dotnet new, restore, build a new app using the sdk image
+            List<string> args = new List<string>();
+            args.Add($"netcoreapp_version={netcoreappVersion}");
+            if (!sdkImageVersion.StartsWith("1."))
```

Comment:
This logic can be pulled into a separate method `public string GetOptionalBuildArgs(string version) { }` since it gets used in SCD test too.

Reply:
The logic in SCD is slightly different.  I didn't see a clear way to share the code that was beneficial.